server/sonar-web/src/main/js/components/workspace/WorkspaceRuleTitle.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { RuleDescriptor } from './context';
+
+interface Props {
+  rule: RuleDescriptor;
+}
+
+export default function WorkspaceRuleTitle({ rule }: Props) {
+  return (
+    <>
+      <i className="icon-workspace-doc little-spacer-right" />
+      <span className="text-limited">{rule.name || '—'}</span>
```

Comment:
It should not be `text-limited` when the workspace is opened. Or the size limit should be bigger.